docs/aeternity-erlang-compatibility.md

```diff
@@ -340,3 +340,11 @@ iex> call_id = Call.id(pub_key, prev_nonce, contract_id)
 iex> call_tree_id = CallStateTree.construct_call_tree_id(contract_id, call_id)
 iex> CallStateTree.get_call(Chain.chain_state.calls, call_tree_id)
 ```
+
+- ```call_data``` (for Solidity ABI) is the first (left, high-order in big-endian) four bytes of the Keccak-256 (SHA-3) hash of the signature of the function
+
+- in order to call a contract function (for Solidity ABI), the hash of the function must be provided, followed by its arguments, encoded as specified in the [spec]( https://solidity.readthedocs.io/en/develop/abi-spec.html#argument-encoding)
+
+- ```code``` is the byte code of the contract. Contracts can be compiled and have their byte code and function signature hashes reviewed, using [Remix IDE](https://remix.ethereum.org/)
+
+- ```call_stack``` is used internally and should be empty when executing from top level
```

Comment:
maybe reference that this is the storage element of each individual contract

Reply:
This is the call stack array, populated only when there are nested calls.

Reply:
okay, so maybe explain this here.